src/oneD/StFlow.cpp

```diff
@@ -293,31 +293,41 @@ void StFlow::_finalize(const double* x)
     }
 }
 
-void StFlow::eval(size_t jg, double* xg, double* rg, integer* diagg, double rdt)
+void StFlow::eval(size_t j_global, double* x_global, double* rsd_global, integer* diag_global, double rdt)
 {
```

Comment:
Ah, I got mixed up in the contributing guide when it linked to the Google code guide.

Reply:
This integer* in the header and the implementation are necessary/different from the standard int?

Reply:
The naming is a bit confusing because there's no real mention of the global arrays in the Domain1D.h docstring for the eval() method, but in the StFlow.h docstring/implementation we had the jg/xg/diagg, etc. and in the implementation is looks like it uses the input variables with loc() to find the local numbering of the global arrays.